docs/Makefile

```diff
@@ -60,7 +61,7 @@ clean:
 cpydiff:
 	@echo "Generating MicroPython Differences."
 	rm -f $(GENRSTDIR)/*
-	cd $(CPYDIFFDIR) && python $(CPYDIFF)
+	cd $(CPYDIFFDIR) && $(PYTHON) $(CPYDIFF)
```

Comment:
Just wondering, would it also not be possible to use the environment variable MICROPY_CPYTHON3 here?